.github/workflows/setup-varnish.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+# Install Varnish
+echo "### Installing Varnish $VARNISH_VERSION ###"
+sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 45DACFFDB8AFA6AA
```

Comment:
even additionally installing this key does not fix the problem.

Reply:
ah, the sources file from varnish packagecloud specifies the location of the gpg key to a specific file, so adding it to the default keyring does nothing.